pr_agent/algo/ai_handler.py

```diff
@@ -55,7 +55,7 @@ def deployment_id(self):
 
     @retry(exceptions=(APIError, Timeout, TryAgain, AttributeError, RateLimitError),
            tries=OPENAI_RETRIES, delay=2, backoff=2, jitter=(1, 3))
-    async def chat_completion(self, model: str, temperature: float, system: str, user: str):
+    async def chat_completion(self, model: str, system: str, user: str, temperature: float = 0.2):
```

Comment:
**Suggestion:** Consider reordering the parameters of the 'chat_completion' function to match the order in the original function. This can help maintain consistency and avoid confusion.
```suggestion
    async def chat_completion(self, model: str, temperature: float, system: str, user: str):
```